docs/docs/guides/core-types/evaluation_playground.md

```diff
@@ -0,0 +1,109 @@
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Evaluation Playground
+
+The W&B Weave Evaluation Playground is a powerful tool in the W&B UI that allows you to access existing models, build new ones, and compare their performances using evaluation datasets and LLM scoring judges. This enables you to start experimenting with and comparing your models without setting up any code. You can also save the models, scorers, and datasets you develop in the playground for later development and deployment.
```

Comment:
I have noticed the use of `W&B` (no weave) and `W&B Weave` throughout this document. for example "Open the W&B UI". It is possible that the docs team is making an effort to remove the concept of `Weave` from docs, but this does seem like a deviation from the norm. Everything should be in terms of `Weave` & `Weave UI`. occassionally, `W&B Weave` when we want to harp on branding, but rarely if ever no `Weave` and just `W&B`